docs/participate/run-nodes/full-node.mdx

```diff
@@ -188,17 +121,19 @@ Download and run the installation script from the **non-root** user account with
 * `-d` - **mytonctrl** will download a [dump](https://dump.ton.org/) of the latest blockchain state.
 This will reduce synchronization time by several times.
 * `-c <path>` - If you want to use not public liteservers for synchronization. _(not required)_
+* `-i` - Ignore minimum requirements, use it only if you want to check compilation process without real node usage.
 
+There are two installation modes: lite and full. They both compile and install TON components. However the lite version does not configure or run the node/validator.
```

Comment:
Did not find any usage of validator flag, only three uses of lightserver -- affects ttl, mode enable function (have searched from mytoninstaller.py, mytonctrl2_dev branch)

Am I missing something or it is just not ready?

Reply:
@Full-Hat currently `validator` flag is just default installation with `validator` mode enabled, but it can be changed in future. it's important that people install mtc in the mode in which they will use it, so we need to emphasize this point